README.md

```diff
@@ -51,5 +51,15 @@ Student number - 13,  HW i-face - UART, Game - chrome dragon game, Data driven f
     До даної версії було зроблені unit test, покрито практично 90% всього проекту, включаючи в себе guards & services.
     Щоб запустити тестування потрібно у терміналі прописати ng test --code-coverage і щщоб побачити шо і як покрито, то відкриваєте через гугл оцей файл cstd-kolodiibohdan-2122\coverage\chrome-dragon-game\index.html і побачите всю картину тестів
 
-                            Kindergarten v1.25  -- added Kindergarten-list & kindergarten-details & updated tests
+                            Kindergarten v1.1  -- added Kindergarten-list & kindergarten-details & updated tests
     До даної версії було додано дві нові компоненти : Kindergarten-list & kindergarten-details. Тепер таба "Kindergarten", котра знаходиться справа в меню працює. Ця таба перенаправить вас на вкладку Kindergarten-list де розпарсаються дані з колекції, в якій міститься всі садочки. Якщо навести курсор мишки на будь-який садочок, то спрацює ховер ефект і ви побачите коротку інформацію про садочок і кнопку, котра перенаправить вас на вкладку з деталями про цей конкретний садочок. На вкладці деталей ви будете бачити тайтл садочку і кнопку(кнопка перенаправить вас вниз сторінки де буде форма, також кнопка заповнить дані про вас(поки тільки імейл)). Якщо поскролити нижче то можна побачити опис про садочок, види груп, переваги садочку, адреси садочку і форми перебування в садочку. В самому низу знаходиться форма, котру потрібно заповнити, якщо хочете подати за'явку на запис вашої дитини у цей садочок. Після того,як ви заповнете форму, то вона буде надіслана в нову колекцію де будуть зберігатися всі форми(коли буде розроблено сторона представника садочку, то він зможе бачити всі ці форми і приймати рішення чи написати вам). До всіх компонент вже розроблено адаптивність(тобто під будь-яку ширину екрану) і оновлено тести. 
+
```

Comment:
What should I review?

Reply:
which type file is respocible for DB from PR#2?

Reply:
In this readme file last line I wrote which files are respocibled for DB